app/Http/Controllers/NoteController.php

```diff
@@ -29,6 +30,40 @@ public function index(Request $request)
         ]);
     }
 
+    public function getNotesByUsername()
+    {
+        $user = Auth::user();
+
+        if (!$user) {
```

Comment:
Why are you always everywhere checking if user exists? It is behind middleware `auth`, so, yes it exists. Despite of that ... proper way of checking if user is present if `Auth::check()`.